packages/qwik-city/runtime/src/client-navigate.ts

```diff
@@ -40,8 +40,12 @@ export const newScrollState = (): ScrollState => {
   };
 };
 
-export const dispatchPrefetchEvent = (prefetchData: QPrefetchData) => {
+export const prefetchSymbols = (path: string) => {
   if (isBrowser) {
-    document.dispatchEvent(new CustomEvent('qprefetch', { detail: prefetchData }));
+    path = path.endsWith('/') ? path : path + '/';
```

Comment:
You are assuming that all paths end with `/` This is the default but it is possible to configure Qwik without trailing slash.